### ISSUE TITLE
Ability to modify the size of free labels (comments)

### ISSUE THREAD

**Issue by RobustoSystems** [HunterJKI/decoration-properties#4]
Hi Hunter,

I hope you are well.

First, thank you for an excellent tool! I learned about it recently, after watching the YouTube recording of your "The LabVIEW UI of Your Dreams" GLA Summit 2024 presentation. I am currently working on a project with a few complex Windows GUIs. The decoration properties tool, as well as the tips you shared in the presentation, are extremely useful.

### The GitHub Issue
It would be great if the tool was able to modify **the size** of free labels (comments) too. The tool can currently modify only the position of free labels.

Currently I'm getting the following Object Properties page for a free label. I get the same behaviour on both the front panel and the block diagram. Using LabVIEW 2024 Q1.
![1](https://github.com/HunterJKI/decoration-properties/assets/34945974/82cb0c59-9575-4da1-9e3f-fd792ea7346e)

Kind regards,
Petru Tarabuta, Robusto Systems, UK

**Comment by HunterJKI** [comment]
Good idea, I stumbled across the ability to use 4.txt for decorations properties, if we can figure out the same mapping for labels it should be pretty copy-paste. If you figure it out please submit a pull request!
